binr/rafind2/rafind2.c

```diff
@@ -203,11 +211,51 @@ static int rafind_open(char *file) {
 	return 0;
 }
 
+static int rafind_open_dir(char *dir) {
+	RList *files;
+	RListIter *iter;
+	void *current_entry;
+	char *filename = NULL;
+	char *fullpath;
+
+	files = r_sys_dir (dir);
+
+	if (files) {
+		r_list_foreach (files, iter, current_entry) {
+			filename = current_entry;
+
+			/* Filter-out unwanted entries */
+			if (strcmp (".", filename) == 0) {
```

Comment:
use ! instead of == 0, and you can compare both . and .. in here. also.. what do you think about just checking if the first char is '.' instead? hidden files are usually something to skip